src/proxyClient.js

```diff
@@ -97,7 +97,14 @@ var frameId = 0;
 
 // Worker
 
-var worker = new Worker('{{{ filename }}}.js');
+var filename = '{{{ filename }}}.js';
+var workerURL = filename;
+var fileBytes = tryParseAsDataURI(filename);
+if (fileBytes) {
+  workerURL = URL.createObjectURL(new Blob([fileBytes], {type: 'application/javascript'}));
+}
+
+var worker = new Worker(workerURL);
```

Comment:
> Alternatively, I could remove the deprecation commit and submit it as a separate PR for greater visibility and/or dedicated discussion among all interested parties.

I think something like this would be best. There is no fundamental reason why we should deprecate that feature as part of this pull request, and we should probably ask out if it is being used. It does have good performance, on average better than base64, so it does have merit.